.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: build
+on:
+  push:
+    branches:
+      - '*'
+    tags-ignore:
+      - '*'
+  pull_request:
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest,macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Pre-reqs (apt)
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt-get -y update && sudo apt-get install build-essential \
+            libpng-dev zlib1g-dev libx11-dev libxaw7-dev libxt-dev x11proto-core-dev libgif-dev
+      - name: Pre-reqs (homebrew)
+        if: runner.os == 'macOS'
+        run: |
+          brew reinstall gfortran
+          brew install automake \
+            libpng \
+            libxt
+      - name: Bootstrap
+        shell: bash
```

Comment:
Yep, the defaults are OK. I'm just doing some overly defensive programming here since I got bit with an issue where I was using `bash` syntax when working on other workflows.